src/styles/index.scss

```diff
@@ -1,3 +1,147 @@
 body {
-  margin: 0;
+  $card-border: #f3f3f3;
+  $color-black: #060b35;
+  $color-gray: #616070;
+  $color-blue: #00acdc;
+  $color-white: #fff;
+  $gap: 4px;
+
```

Comment:
You can use import to import variables from another file.
Also, it's better to split the index file into 
- resets (tags styles and resets here for body, *, etc.)
- variables
- mixins
- stars
- card